package.json

```diff
@@ -12,11 +12,12 @@
     "color": "#0072c6",
     "theme": "dark"
   },
+  "enableProposedApi": true,
```

Comment:
Will require allow-listing the extension id in the product.json.

Reply:
Have added it there 👍